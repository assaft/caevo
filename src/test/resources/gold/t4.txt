John left 2 days ago.