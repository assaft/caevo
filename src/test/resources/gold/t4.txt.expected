ei1,left
t1,1969-07-19
l1,ei1,t1,IS_INCLUDED